### ISSUE TITLE
Page selector doesn't work anymore in {N} 2.4

### ISSUE THREAD

**Issue by jboeijenga** [NativeScript/NativeScript#3117]
After upgrading our version of Nativescript to 2.4 the Page selector we have in our app.css does not seem to work anymore. 
We use the Page selector to set the default font-family font-size and color of the whole application (including ActionBar)

**Comment by tsonevn** [comment]
Hi @jboeijenga,
I reviewed your problem, however was to reproduce such a behavior with the latest **modules** and **CLI**. To test this behavior, I created new project using `tns create <project name>` and use the following code. 
**main-page.xml**
```
<Page xmlns="http://schemas.nativescript.org/tns.xsd" navigatingTo="navigatingTo">
    <StackLayout class="p-20">
        <Label text="Tap the button" class=""/>
        <Button text="TAP" tap="{{ onTap }}" class="btn btn-primary btn-active"/>
        <Label text="{{ message }}" class="" textWrap="true"/>
    </StackLayout>
</Page>
```

**app.css**
```
Page{
    color:red;
    font-size: 12;
}
.btn {
    font-size: 18;
}
```
It would help if you could provide sample project, which could be debugged locally.

Regards,
@tsonevn

**Comment by jboeijenga** [comment]
Hi @tsonevn ,

Thanks for the reply. 
I've created a new Angular2 project and saw that indeed the Page selector is working fine.

I have also found out that in my application the whole app.css file is not applied correctly.
Maybe that's due to the fact that we have restructured some folders.

From the documentation however it's now not clear to me anymore how to change the location for the application wide css file
[https://docs.nativescript.org/angular/ui/styling.html#application-wide-css](https://docs.nativescript.org/angular/ui/styling.html#application-wide-css) is not the way to go anymore in 2.4. 

Can someone explain more about this?

**Comment by tsonevn** [comment]
Hi @jboeijenga,
Excuse me for the delay in replying.

The correct way to change the location for the application wide css is by using `platformNativeScriptDynamic({bootInExistingPage:false, cssFile:"style.css"});` method. I review the attached link and found that we missed to update the article about the styling in NativeScript Angular 2.  I upgrade this article with this pull request and the changes will soon be available in the web site. In my further research I also found that setting `cssFile` property will not be respect correctly in the application. Regarding to that we also managed to fix this problem with PR [#557](https://github.com/NativeScript/nativescript-angular/pull/557) .

Hope this information helps.

**Comment by jboeijenga** [comment]
@tsonevn thanks for the reply.
I also found out that not all AppOptions like cssFile are working correctly.
As a workaround I now put 
`application.cssFile = '<CUSTOM_PATH>';`
before 
`platformNativeScriptDynamic().bootstrapModule(AppModule);`
That is working correctly for me.
